{{ cookiecutter.package_name }}/setup.cfg

```diff
@@ -1,62 +1,74 @@
 [metadata]
 name = {{ cookiecutter.module_name }}
-# version should be PEP440 compatible (https://www.python.org/dev/peps/pep-0440/)
-version = {{ cookiecutter.project_version }}
 author = {{ cookiecutter.author_name }}
 author_email = {{ cookiecutter.author_email }}
-description = {{ cookiecutter.short_description }}
-long_description = {{ cookiecutter.long_description }}
 license = {{ cookiecutter.license }}
+license_file = licenses/LICENSE.rst
 url = {{ cookiecutter.project_url }}
+description = {{ cookiecutter.short_description }}
+long_description = file: README.rst
 edit_on_github = {{ cookiecutter.edit_on_github_extension }}
 github_project = {{ cookiecutter.github_project }}
-python_requires = ">={{ cookiecutter.minimum_python_version }}"
 
 [options]
-# install_requires should be formatted as a semicolon-separated list, e.g.:
-# install_requires = astropy; scipy; matplotlib
-install_requires = {{ cookiecutter._install_requires }}
 zip_safe = False
-use_2to3 = False
+packages = find:
+python_requires = >={{ cookiecutter.minimum_python_version }}
+setup_requires = setuptools_scm
+install_requires = {{ cookiecutter._install_requires }}
 
+{% if cookiecutter.include_example_code == 'y' %}
 [options.entry_points]
 console_scripts =
-    {% if cookiecutter.include_example_code == 'y' %}
     astropy-package-template-example = packagename.example_mod:main
-    {% else %}
-    # astropy-package-template-example = packagename.example_mod:main
-    {% endif %}
+{% endif %}
+
+[options.extras_require]
+all =
```

Comment:
Why the empty `all=`?

Reply:
It's easier this way to show where optional dependencies should go.

Reply:
can you leave a comment for it then?

Reply:
also, one could have the deps listed in the cookiecutter template?

Reply:
Do you mean this should be a question cookiecutter asks?

Reply:
yes, but it's not a strong preference, just along the line of the logic that we otherwise use the cookiecutter to fill out these boilerplates all the other places.

Reply:
I've now added this - both for required and optional dependencies